dandi/download.py

```diff
@@ -1143,17 +1161,9 @@ def _check_if_more_attempts_allowed(
                 sleep_amount,
                 exc,
             )
-        else:
-            lgr.debug(
-                "%s - download failed on attempt #%d: %s, will sleep a bit and retry",
-                path,
-                attempt,
-                exc,
-            )
-    # if is_access_denied(exc) or attempt >= 2:
-    #     raise
-    # sleep a little and retry
-    else:
```

Comment:
By removing this `else`, the `lgr.debug()` call on line 1138/1156 will be followed by the call on line 1157/1167, resulting in two similar log messages.

Reply:
i think it would only be  in the case of non-conformant "retry-after" we would get 3 messages "on the matter"
- "to response %d with non-integer or future date"
- "to response %d with Retry-After"
- the one below.

if there is a good Retry-after, we would get a non-negative value for sleep_amount, and log only at 1138/1156 (but we might want it into `else:` for exception, to not log with /1147) as sleep_amount condition on /1164 would not trigger that other message where we would also define the random sleep.

But for above "3 msgs" I will adjust... thanks!